recipes/vc-feature/meta.yaml

```diff
@@ -0,0 +1,23 @@
+{% set version = "0.0.1" %}
+
+package:
+  name: vc-feature
+  version: {{ version }}
+
+build:
+  number: 0
+  track_features:
+    - vc9     # [win and py27]
+    - vc10    # [win and py34]
+    - vc14    # [win and py35]
+  skip: true  # [unix]
```

Comment:
Because we need a dependency during build time to activate the features. That is why we have used Python in the past. The hope though is that once we do that one hack here. We will never need to do it anywhere else again.